code/obj/machinery/computer/cloning.dm

```diff
@@ -550,17 +550,7 @@ proc/find_ghost_by_key(var/find_key)
 			boutput(user, "<span class='notice'><B>The scanner is already occupied!</B></span>")
 			return
 
-		var/mob/M = G.affecting
-		M.set_loc(src)
-		src.occupant = M
-		src.icon_state = "scanner_1"
-
-		playsound(src.loc, "sound/machines/sleeper_close.ogg", 50, 1)
-
-		for(var/obj/O in src)
-			O.set_loc(src.loc)
-
-		src.add_fingerprint(user)
+		move_mob_inside(G.affecting)
```

Comment:
Looks like `move_mob_inside` checks for `can_operate`, which returns false if someone is out of bounds or incapacitated. I think this means that you wouldn't be able to grab people and shove them inside if they were passed out/not directly adjacent to the scanner/etc.? Would recommend modifying `move_mob_inside` so that it's passed both a `user` and a `M` var.